src/scss/patterns/_offset-grid.scss

```diff
@@ -43,3 +43,24 @@ $break: 'wide-break';
     padding-left: 0;
   }
 }
+
+//services page
+
+.offset-media {
+  --offset-media-padding: 0;
+  display: block;
+  padding: var(--offset-media-padding);
+
+  &:nth-child(odd) {
+    padding: var(--offset-media-padding);
+  }
+
+  &:nth-child(even) {
+    padding: var(--offset-media-padding);
+  }
+
+  img {
+    height: 100%;
```

Comment:
Might get this by default by using `flex` or `grid` on the container, rather than setting the explicit height here? This works great, so not a real concern – just a slight preference for more implicit approaches where possible in CSS.

Reply:
setting the container to `flex` and setting `flex: 1 0 auto` on the items appears to work to get the images in the right place. 

I also added `object-position` to the images to get them centered in a better way. You don't fall off the screen anymore!